gateway/ln-gateway/src/lib.rs

```diff
@@ -500,10 +513,15 @@ impl Gateway {
             lightning_alias,
         } = self.state.read().await.clone()
         {
+            let gateway_config = self
+                .get_gateway_configuration()
+                .await
+                .expect("Gateway configuration should be set");
```

Comment:
should we gracefully exit the api call instead?

Reply:
It should be guaranteed to exist at this point. `get_gateway_configuration` should only return `None` if we're in the `Configuring` state. I'll add a comment